content/200-concepts/100-components/02-prisma-client/053-client-extensions/199-extension-examples.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: 'Extension examples'
```

Comment:
Why not just call it "Examples"? 

"Extension" is redundant here because we're already in the "Prisma Client Examples" section.